keyboards/rad/Makefile

```diff
@@ -0,0 +1,16 @@
+RAD

```

Comment:
This file looks like it's supposed to be a readme, and would need to be renamed to `readme.md`